Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG RUST_IMAGE=rust:1.42.0-buster
+ARG RUNTIME_IMAGE=debian:buster-slim
+
+FROM $RUST_IMAGE as build
+RUN rustup component add rustfmt
```

Comment:
Why do we need rustfmt to build a docker image?

Reply:
tonic

Reply:
oh. right.

Reply:
looks like we can also just [turn off the `rustfmt` thing](https://docs.rs/tonic-build/0.2.0/tonic_build/struct.Builder.html#method.format) — would that be better?